tests/sentry/issues/endpoints/test_organization_group_suspect_tags.py

```diff
@@ -61,7 +62,14 @@ def test_get_no_group(self) -> None:
             response = self.client.get("/api/0/issues/22/suspect/tags/")
             assert response.status_code == 404
 
-    def _mock_event(self, ts, hash="a" * 32, group_id=None, project_id=1, tags=None):
+    def _mock_event(
+        self,
+        ts: datetime.datetime,
+        tags: dict[str, Any],
+        hash: str = "a" * 32,
+        group_id: int | None = None,
+        project_id: int = 1,
```

Comment:
From the only uses if this I see in the test file, I think these can be simplified.

```suggestion
        hash: str,
        group_id: int,
        project_id: int,
```